app/client/src/routes/helpdesk.tsx

```diff
@@ -0,0 +1,258 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import icon from "uswds/img/usa-icons-bg/search--white.svg";
+import icons from "uswds/img/sprite.svg";
+// ---
+import { serverUrl, fetchData } from "../config";
+import { useHelpdeskAccess } from "components/app";
+import Loading from "components/loading";
+import Message from "components/message";
+import MarkdownContent from "components/markdownContent";
+import { TextWithTooltip } from "components/infoTooltip";
+import { useUserState } from "contexts/user";
+import { useContentState } from "contexts/content";
+import { useDialogDispatch } from "contexts/dialog";
+
+type SubmissionState =
+  | {
+      status: "idle";
+      data: null;
+    }
+  | {
+      status: "pending";
+      data: null;
+    }
+  | {
+      status: "success";
+      data: {
+        // NOTE: more fields are in a form.io submission,
+        // but we're only concerned with the fields below
+        _id: string;
+        state: "submitted" | "draft";
+        modified: string;
+        data: {
+          applicantOrganizationName: string;
+          last_updated_by: string;
+          // (other fields...)
+        };
+        // (other fields...)
+      };
+    }
+  | {
+      status: "failure";
+      data: null;
+    };
+
+export default function Helpdesk() {
+  const navigate = useNavigate();
+
+  const [searchText, setSearchText] = useState("");
+  const [formId, setFormId] = useState("");
+
+  const { epaUserData } = useUserState();
+  const { content } = useContentState();
+  const dispatch = useDialogDispatch();
+  const helpdeskAccess = useHelpdeskAccess();
+
+  const [rebateFormSubmission, setRebateFormSubmission] =
+    useState<SubmissionState>({
+      status: "idle",
+      data: null,
+    });
+
+  if (
+    epaUserData.status !== "success" ||
+    helpdeskAccess === "idle" ||
+    helpdeskAccess === "pending"
+  ) {
+    return <Loading />;
+  }
+
+  if (helpdeskAccess === "failure") {
+    navigate("/", { replace: true });
+  }
```

Comment:
Here's what prevents the user from accessing this page if they shouldn't have access.

@dpgalloway again, just an FYI.